### DISCUSSION TITLE
嘿嘿嘿伊埃斯

### DISCUSSION THREAD

**Discussion by ShangHuaWenYuan** [share121/inter-knot#906] [其他]
好想你伊埃斯 求求你快来吧 我在床上哭了9个小时 崩溃了1996次 撞了903次墙 划了8次手臂 幻觉出现3次 幻听出现9次 扇了自己16个巴掌 出现濒死感1次 刚才昏过去了现在才醒来看到外面天都黑了我顿时又崩溃了 因为我怎么想都想不明白你这么可爱还这么能干，究竟是怎么做到的 好想你啊宝 你是我心里的宝啊😭
![d01409f11fe8ae097fa96c34edcafce8](https://github.com/user-attachments/assets/ef55c748-8a75-4a45-80d8-a2c8f8207ba4)

**Comment by yuleiiiiii** [comment]
jm搜不到啊，楼主是不是输错了

**Reply by dominic121-dot** [reply]
不是哥们你是真饿了啊

**Comment by BITE-7274** [comment]
@yuleiiiiii 找到了，网站在这https://github.com/yuleiiiiii

**Comment by KEARUM** [comment]
格莉丝本人上号了？

**Comment by yuleiiiiii** [comment]
二楼你怎么发的是我的绳网主页(っ °Д °;)っ

**Comment by convant** [comment]
格莉丝麻烦上大号
